test-data/unit/check-incremental.test

```diff
@@ -5586,3 +5586,75 @@ python_version=3.6
 [out]
 [out2]
 tmp/a.py:2: error: Incompatible types in assignment (expression has type "int", variable has type "str")
+
+[case testBaseClassPluginHookWorksIncremental]
```

Comment:
I'd move this to `check-custom-plugin.test` and move the plugin code to a separate file.

Reply:
OK, makes sense.